test/stateful_dataloader/test_state_dict.py

```diff
@@ -1441,6 +1444,154 @@ def test_fast_state_dict_request_skip_steps(self) -> None:
         self._run_test(17, 19)
 
 
+class TestMultiEpochSDL_shard0(TestCase):
+    def get_map_dl(self, data_size=100, num_workers=0, batch_size=1, shuffle=False):
```

Comment:
can you test this for num_workers=0 and say num_workers=2?

Reply:
Removed defaults values for `num_workers`.
Tests below run for both 0 and 2.